src/network/networkd-address-label.h

```diff
@@ -19,6 +19,7 @@ typedef struct NetworkConfigSection NetworkConfigSection;
 struct AddressLabel {
         Network *network;
         NetworkConfigSection *section;
+        bool section_is_dirty;
```

Comment:
hmm, is "dirty" really the right word for this? that term is usually used to mark cache entries that need to be resynced, but that's not the case here. Maybe just call this "section_invalid"?